netstandard/ref/System.Runtime.CompilerServices.cs

```diff
@@ -170,6 +184,18 @@ public sealed partial class ConditionalWeakTable<TKey, TValue> : System.Collecti
         public bool TryGetValue(TKey key, out TValue value) { throw null; }
         public delegate TValue CreateValueCallback(TKey key);
     }
+    public readonly partial struct ConfiguredAsyncEnumerable<T>
+    {
+        private readonly object _dummy;
+        public System.Runtime.CompilerServices.ConfiguredAsyncEnumerable<T>.Enumerator GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public readonly partial struct Enumerator
+        {
+            private readonly object _dummy;
+            public T Current { get { throw null; } }
+            public System.Runtime.CompilerServices.ConfiguredValueTaskAwaitable DisposeAsync() { throw null; }
+            public System.Runtime.CompilerServices.ConfiguredValueTaskAwaitable<bool> MoveNextAsync() { throw null; }
+        }
+    }
```

Comment:
This is likely going to change as part of https://github.com/dotnet/corefx/issues/33909.  Maybe don't add it yet?

Reply:
Sounds good. Let's try to close on this next week.